front/src/modules/trainschedule/components/ManageTrainSchedule/TimetableManageTrainSchedule.tsx

```diff
@@ -18,7 +18,7 @@ import { useAppDispatch } from 'store';
 import { isPacedTrain, isTrainSchedule } from 'utils/trainId';
 
 import AddTrainScheduleButton from './AddTrainScheduleButton';
-import useUpdateTimetableItem from './hooks/useUpdateTimtetableItem';
```

Comment:
Oh, nice catch! Took me a few tries of careful char-by-char reading to figure out that these two strings are different and that there was a typo in the previous names.